entities/A_Station.php

```diff
@@ -88,7 +95,7 @@ public static function fetchForHighway($hid) {
    * @return int -1 if not possible, otherwise the ID that the onramp *should* be.
    */
   public static function calculateRelatedOnrampID($tid) {
-    if($tid >= 1000 && $tid < 4000) {
+    if($tid < 4000 && $tid >= 1000 ) {
```

Comment:
I work in mysterious ways ooo000oo000oo

Actually I was trying to debug something and thought maybe it was only checking the first condition as true then moving on but thats only for or statements and I was too lazy to reverse it.